### PR TITLE
Add Cobo wallet

### PR DESCRIPTION
**Pull request by bbbolongx**
Add Cobo wallet to wallet section

### CODE REVIEW

File: _translations/en.yml [hunk 1]
```diff
@@ -185,6 +185,7 @@ en:
...
     walletbitcoinknots: "Bitcoin Knots is a full Bitcoin client and builds the backbone of the network. It offers high levels of security, privacy, and stability. It includes more advanced features than Bitcoin Core, but they are not as well-tested. It uses a lot of space and memory."
+    walletcobo:"Cobo is the first leading wallet company in the world to offer Staking and masternode rewards on user holdings, making it easy for users to grow their digital assets effortlessly. Cobo is the first leading wallet company in the world to offer Staking and masternode rewards on user holdings, making it easy for users to grow their digital assets effortlessly. As a company, we emphasize long-term security, reliability, and convenience. Our key products and services reflect that vision. Altogether, Cobo offers a secure mobile wallet (Cobo Wallet), high-end cold storage wallet for advanced users (Cobo Vault), and custodial services for institutional investors. With support for over thirty different coins, 700 tokens, and an integrated DApp store along the way, we aim to be a one-stop shop for everything crypto."
```
Comment (wbnns) [inline _translations/en.yml]: A space after the colon is missing here.

File: _translations/zh_CN.yml [hunk 2]
```diff
@@ -144,6 +144,7 @@ zh_CN:
     walletcatweb: "网页"
     walletbitcoincore: "比特币核心是一个实现了全节点的比特币客户端，它组成了整个比特币网络的支架。比特币核心拥有极高的安全性、隐私性、稳定性。但是它有较少的特性且会占用很多的磁盘和内存空间。"
     walletbitcoinknots: "比特币的许多节点都是一个完整的比特币客户端，并且建立了比特币网络的主干。客户端提供了高级别的安全、高度隐私和高标准稳定性。客户端要比比特币核心包含更高级的特性，但这些客户端并没有很好的进行测试。客户端会占用大量的磁盘空间和计算机内存。"
+    walletcobo:"Cobo是全球第一家主打Staking和节点增益功能的数字货币钱包，希望通过努力让用户“拥有和使用数字货币变得简单”。Cobo提供长期安全、可靠、便利的产品和服务。Cobo的产品线包括面向初级用户的「Cobo钱包」、面向高端用户的冷存储产品「Cobo金库」以及面向机构投资者的「Cobo 托管」。Cobo支持三十多种主流币种、700多种代币，并汇聚优质DAPP，我们致力于提供一站式数字货币服务。"
```
Comment (wbnns) [inline _translations/zh_CN.yml]: A space after the colon is missing here.

File: _wallets/cobowallet.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+level: 3
+platform:
+  - mobile:
+    name: mobile
+    os:
+      - name: android
+        text: "walletcobo"
+        link: "https://play.google.com/store/apps/details?id=cobo.wallet"
+        source: "https://github.com/cobowallet"
+        screenshot: "##"
```
Comment (wbnns) [inline _wallets/cobowallet.md]: A screenshot is missing here.

File: _wallets/cobowallet.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          fees: "checkpassfeecontroldynamic"
+        privacycheck:
+          privacyaddressreuse: "checkpassprivacyaddressrotation"
+          privacydisclosure: "checkfailprivacydisclosureaccount"
+          privacynetwork: "checkfailprivacynetworknosupporttor"
+      - name: ios
+        text: "walletcobo"
+        link: "https://itunes.apple.com/us/app/cobo-cryptocurrency-wallet/id1406282615"
+        source: "https://github.com/cobowallet"
+        screenshot: "##"
```
Comment (wbnns) [inline _wallets/cobowallet.md]: A screenshot is missing here.

File: _wallets/cobowallet.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+compat: "mobile android ios"
+level: 3
+platform:
+  - mobile:
+    name: mobile
+    os:
+      - name: android
+        text: "walletcobo"
+        link: "https://play.google.com/store/apps/details?id=cobo.wallet"
+        source: "https://github.com/cobowallet"
```
Comment (wbnns) [inline _wallets/cobowallet.md]: A direct link to the repository containing the wallet source code is missing here.

File: _wallets/cobowallet.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+          privacy: "checkpassprivacybasic"
+          fees: "checkpassfeecontroldynamic"
+        privacycheck:
+          privacyaddressreuse: "checkpassprivacyaddressrotation"
+          privacydisclosure: "checkfailprivacydisclosureaccount"
+          privacynetwork: "checkfailprivacynetworknosupporttor"
+      - name: ios
+        text: "walletcobo"
+        link: "https://itunes.apple.com/us/app/cobo-cryptocurrency-wallet/id1406282615"
+        source: "https://github.com/cobowallet"
```
Comment (wbnns) [inline _wallets/cobowallet.md]: A direct link to the repository containing the wallet source code is missing here.

File: _wallets/cobowallet.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+          privacydisclosure: "checkfailprivacydisclosureaccount"
+          privacynetwork: "checkfailprivacynetworknosupporttor"
+  - web:
+    name: web
+    os:
+      - name: web
+        text: "walletcobo"
+        link: "https://cobo.com/"
+        source: "https://github.com/cobowallet"
+        screenshot: "##"
```
Comment (wbnns) [inline _wallets/cobowallet.md]: A screenshot is missing here.

File: _wallets/cobowallet.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+          privacyaddressreuse: "checkpassprivacyaddressrotation"
+          privacydisclosure: "checkfailprivacydisclosureaccount"
+          privacynetwork: "checkfailprivacynetworknosupporttor"
+  - web:
+    name: web
+    os:
+      - name: web
+        text: "walletcobo"
+        link: "https://cobo.com/"
+        source: "https://github.com/cobowallet"
```
Comment (wbnns) [inline _wallets/cobowallet.md]: A direct link to the repository containing the wallet source code is missing here.

**Comment by bbbolongx** [comment]
Hi,.
Should I just fill in the cobowallet.png? I don't why other wallets have
another name with "name+?+number.png"



[image: image.png]
[image: image.png]

**Comment by bbbolongx** [comment]
*@wbnns Please help*

File: _wallets/cobowallet.md (hunk 3 above)
Comment (bbbolongx) [reply _wallets/cobowallet.md]: How can i use the path for screenshot?

**Comment by bbbolongx** [comment]
Finished.

**Pull request closed without merging**